core/src/main/resources/static/index.html

```diff
@@ -414,6 +414,12 @@ <h4 class="mb-3">Shortcuts</h4>
                         </ul>
                     </li>
 
+                                                                  <li>
+                        <a class="mdi mdi-grease-pencil" target="_blank" href={{dashboardDetails.googleFeedbackFormLink}}>
```

Comment:
I know this is painful but I think we should add rel="noreferrer noopener"

Its a little overkill because we control the link but if it is good practice.

more details here 
https://www.elegantthemes.com/blog/wordpress/rel-noopener-noreferrer-nofollow#:~:text=The%20rel%3D%E2%80%9Dnoopener%20noreferrer%E2%80%9D%20HTML%20attributes%20appear%20in%20external,in%20the%20destination%20site's%20analytics